patches/common/chromium/printing.patch

```diff
@@ -121,7 +152,34 @@ index 007b003dd58d44acd6e1351c237fca6463d90602..4ddbf18e335609623aba42cc39a7c977
  #endif
  
    ReleasePrinterQuery();
-@@ -594,6 +599,9 @@ void PrintViewManagerBase::ReleasePrintJob() {
+@@ -427,6 +433,7 @@ void PrintViewManagerBase::Observe(
+ void PrintViewManagerBase::OnNotifyPrintJobEvent(
+     const JobEventDetails& event_details) {
+   switch (event_details.type()) {
++    case JobEventDetails::USER_INIT_CANCELED:
+     case JobEventDetails::FAILED: {
+       TerminatePrintJob(true);
+ 
+@@ -437,8 +444,7 @@ void PrintViewManagerBase::OnNotifyPrintJobEvent(
+       break;
+     }
+     case JobEventDetails::USER_INIT_DONE:
+-    case JobEventDetails::DEFAULT_INIT_DONE:
+-    case JobEventDetails::USER_INIT_CANCELED: {
++    case JobEventDetails::DEFAULT_INIT_DONE: {
+       NOTREACHED();
+       break;
+     }
+@@ -533,7 +539,7 @@ bool PrintViewManagerBase::CreateNewPrintJob(PrinterQuery* query) {
+   DCHECK(query);
+ 
+   // Disconnect the current |print_job_|.
+-  DisconnectFromCurrentPrintJob();
++  // DisconnectFromCurrentPrintJob();
```

Comment:
we don't want to call this twice, since it prevents `printing_succeeded_` from ever being set to true and thus caused https://github.com/electron/electron/issues/17265